### PR TITLE
Circleci 2.0

### PR DESCRIPTION
**Pull request by jarell-lloyd**

### CODE REVIEW

**Comment by jarell-lloyd** [comment]
- [ ] Review config file

- [ ] compare to old file to ensure everything lines up (folders, git repos, job names, etc.)

- [ ] build passes https://circleci.com/gh/GetAmbassador/redux-clerk/267

**Comment by jarell-lloyd** [comment]
@will-hitchcock please review. thanks

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    - run: if [ -z "${NODE_ENV:-}" ]; then export NODE_ENV=test; fi
+    - run: export PATH="~/GetAmbassador/redux-clerk/node_modules/.bin:$PATH"
+    - run: npm install
+    # Save dependency cache
+    - save_cache:
+        key: v1-dep-{{ .Branch }}-{{ epoch }}
+        paths:
+        # This is a broad list of cache paths to include many possible development environments
+        # You can probably delete some of these entries
+        - vendor/bundle
```
Comment (will-hitchcock) [inline .circleci/config.yml]: I think we can probably delete some of these?

**Comment by jarell-lloyd** [comment]
New build https://circleci.com/gh/GetAmbassador/redux-clerk/269

**Pull request merged**